src/styles/card.scss

```
@@ -0,0 +1,99 @@
+@import 'variables';
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
```

Make a variable for all colors
```suggestion
  border: 1px solid $c-someColor;
```